docs/migrating-to-velero.md

```diff
@@ -0,0 +1,50 @@
+# Migrating from Ark to Velero
+
+Users of Ark will need to [upgrade to v0.10.x][1], then move their installation to the renamed Velero images and Custom Resource Definitions (CRDs), as well as switch to the `velero` client. This is a one-time operation.
+
+At a high level, this process involves the following steps:
+
+    * Scale down the `ark` deployment, so it will not process schedules, backups, or restores during the migration period.
+    * Create a new namespace (named `velero` by default).
+    * Apply the new CRDs into the new namespace.
```

Comment:
The CRD definitions are cluster-scoped so they don't go "into the new namespace". They *are* for the new API group though.